Specs/DateTime/German/DateTimeModel.json

```diff
@@ -3616,5 +3616,392 @@
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2019-06-01",
+              "type": "date",
+              "value": "2019-06-01"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Ich besuche dich an Augsburger Friedensfest letzten Jahres" ,
+    "Context": {
+      "ReferenceDateTime": "2018-11-07T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "augsburger friedensfest letzten jahres",
+        "Start": 20,
+        "End": 57,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2017-08-08",
+              "type": "date",
+              "value": "2017-08-08"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Ich besuche dich an Johannistag" ,
+    "Context": {
+      "ReferenceDateTime": "2018-11-07T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "johannistag",
+        "Start": 20,
+        "End": 30,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "XXXX-06-24",
+              "type": "date",
+              "value": "2018-06-24"
+            },
+            {
+              "timex": "XXXX-06-24",
+              "type": "date",
+              "value": "2019-06-24"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Ich besuche dich an Peter und Paul " ,
+    "Context": {
+      "ReferenceDateTime": "2018-11-07T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "peter und paul",
+        "Start": 20,
+        "End": 33,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "XXXX-06-29",
+              "type": "date",
+              "value": "2018-06-29"
+            },
+            {
+              "timex": "XXXX-06-29",
+              "type": "date",
+              "value": "2019-06-29"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Ich besuche dich an Allerheiligen" ,
+    "Context": {
+      "ReferenceDateTime": "2018-11-07T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "allerheiligen",
+        "Start": 20,
+        "End": 32,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "XXXX-11-01",
+              "type": "date",
+              "value": "2018-11-01"
+            },
+            {
+              "timex": "XXXX-11-01",
+              "type": "date",
+              "value": "2019-11-01"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Ostern ist dieses Jahr früh" ,
```

Comment:
Is "early this year" or "earlier this year" supported in English?

Reply:
yes both are supported.

Reply:
Then please open an issue to also support it in German. As this spec shows, the "early/earlier" part doesn't seem to be supported along with "this year". Thanks.

Reply:
ok, will do.